### DISCUSSION TITLE
Meilisearch 1.10 and 1.11

### DISCUSSION THREAD

**Discussion by hmrst** [beyondcode/herd-community#1025] [General]
Would be great if we could have the latest Meilisearch version in Herd :)

**Comment by esseremmerik** [comment]
Meanwhile, we are having Meilisearch 1.13 as latest release and Herd it still on 1.9.0.
I really like to have the services installed from one place, but if the version updates are that delayed, it makes it less usable.

**Comment by mpociot** [comment]
Yeah, I totally agree with you.
We've been working very hard on a one-click option to update existing services to their latest versions, which will be available very soon!

https://github.com/user-attachments/assets/c73a4e4f-2972-49c0-9386-43901f814e22

**Reply by esseremmerik** [reply]
That is very nice! Looking forward to the update, great job!

**Reply by esseremmerik** [reply]
Hi @mpociot,

Just installed latest version of Herd with the new vertical navigation, but I only see the update option for Reverb (which stays on beta after clicking update). Do I need a reinstall / config change or is this not available for all services yet? We are particular interested in updating Meilisearch as mentioned before. 

Thanks for an update!

**Comment by LeShadow** [comment]
Hey @mpociot ,

Just out of curiosity, is there any update on this item? Is there any timeline for a release of this feature? 😊

Thanks!

**Comment by karlatascent** [comment]
Any update on this? Would really love to be able to update to the latest 😄 

Thanks @mpociot and team for all your hard work!

**Comment by virgildotcodes** [comment]
Happy subscriber to Pro, but it would be really nice if we could use the latest versions of all the services offered. It seems like most are over a year out of date. Any updates would be appreciated!

Edit: Just a note, it seems a decent chunk of threads here are specifically in reference to being able to use newer/different versions of the various services being offered (I count Meilisearch, Redis, MySQL and Typesense on the first page). So it seems like this one feature (being able to update to the latest version of a service) would address what's on a number of people's wishlists.
